docs/feature-flags.md

```diff
@@ -132,6 +132,7 @@ This lists the feature flags and their sub-configurations to enable/disable and
         - `providers-config-file` [Required]: The path to the file containing a list of supported cloud providers that the service can provision dataplane clusters to (default: `'config/provider-configuration.yaml'`, example: [provider-configuration.yaml](../config/provider-configuration.yaml)).
         - `cluster-compute-machine-type` [Optional]: The compute machine type to be used for provisioning a new dataplane cluster (default: `m5.2xlarge`).
         - `cluster-openshift-version` [Optional]: The OpenShift version to be installed on the dataplane cluster (default: `""`, empty string indicates that the latest stable version will be used). 
+        - `DYNAMIC_SCALING_CONFIG`: The configuration file that contains information about each Kafka instance types, dynamic scaling configuration. Defaults to `"{developer: {reserved_streaming_units: 3, compute_nodes_config: {max_compute_nodes: 3}}, standard: {reserved_streaming_units: 10, compute_nodes_config: {max_compute_nodes: 9}}}"`
```

Comment:
Instead of the service template parameter, this should document the `dynamic-scaling-config-file` flag instead. The default value would be `config/dynamic-scaling-configuration.yaml` and you can point to the dynamic-scaling-configuration.yaml file as an example. I believe this is required when `auto` scaling is enabled.

See the description of `providers-config-file` above as an example.